contribs/gmf/externs/gmfx.js

```diff
@@ -511,3 +514,36 @@ gmfx.TimePropertyResolutionEnum = {
  * }}
  */
 gmfx.TimeProperty;
+
+
+/**
+ * @typedef {{
+ *  data: Object,
```

Comment:
I define later `gmfx.ShortenerAPIResponse.data`  as `@type {{ short_url : string }}` line 532.
I've never seen a record used inside a typedef but I'll try it